runtime/reconcilers/alert.go

```diff
@@ -896,15 +923,35 @@ func extractQueryResultFirstRow(q runtime.Query) (map[string]any, bool, error) {
 			res := make(map[string]any)
 			res[q.DimensionName] = row.DimensionValue
 			for _, v := range row.MeasureValues {
-				res[v.MeasureName] = v.BaseValue.AsInterface()
+				var measureLabel string
+				var f formatter.Formatter
+				for _, measure := range measures {
+					if v.MeasureName == measure.Name {
```

Comment:
Same comments as above about a util func (probably a different one since the logic is quite different) and maybe falling back to non-formatted measures instead of failing